CVs/input4MIPs_source_id.json

```diff
@@ -1091,6 +1119,26 @@
         "mip_era":"CMIP7",
         "source_version":"2.0.0"
     },
+    "UOEXETER-CMIP-2-2-1":{
```

Comment:
Same issue - this has not only the biomass burning but also volcanic data - a conflict, no